api4/config.go

```diff
@@ -24,6 +24,7 @@ type filterType string
 
 const filterTypeWrite filterType = "write"
 const filterTypeRead filterType = "read"
+const featureFlagsElementName string = "FeatureFlags"
```

Comment:
Super duper Nit but the casing for our CONST in server is now `TitleCase`
You can also probably reduce this to something like
```suggestion
const FilterTypeWrite filterType = "write"
const FilterTypeRead filterType = "read"
const FeatureFlagsElementName string = "FeatureFlags"
```
Here is an example of existing const 
![image](https://user-images.githubusercontent.com/17804942/112010543-c7e58980-8afd-11eb-99c9-51ba5bf53709.png)